Tasks/DownloadPackageV1/main.ts

```diff
@@ -35,6 +35,14 @@ async function main(): Promise<void> {
             return Promise.resolve();
         }
 
+        var feedId = projectFeed;
+        var project = null;
+        if(projectFeed.includes("/")) {
```

Comment:
any reason not to use getProjectIdAndFeedIdFromInputParam here?

Reply:
Fixed